common/baselines/librispeech/ls100/gmm/baseline_config.py

```diff
@@ -0,0 +1,57 @@
+"""
+Definition of the pipeline in terms of inputs and steps that are executed
+"""
+from sisyphus import gs
+
+from i6_experiments.common.setups.rasr import gmm_system
+from i6_experiments.common.setups.rasr.util import RasrSteps, OutputArgs
+
+from i6_experiments.common.baselines.librispeech.ls100.gmm import baseline_args
+from i6_experiments.common.baselines.librispeech.data import get_corpus_data_inputs
+
+from ...default_tools import RASR_BINARY_PATH
+
+
+def run_librispeech_100_common_baseline(
+    alias_prefix="baselines/librispeech/ls100/gmm/common_baseline",
+):
+
+    gs.ALIAS_AND_OUTPUT_SUBDIR = alias_prefix
```

Comment:
maybe store the previous value of `gs.ALIAS_AND_OUTPUT_SUBDIR` in a local variable to restore that later.